templates/archives/stories/women_in_symbols.jinja2

```diff
@@ -0,0 +1,39 @@
+
+<figure class="figure float-right">
+    <img src="{{ static('img/stories/women_in_symbols.png') }}"
+         class="class="figure-img img-fluid rounded"
+         alt="A letter subscript above the cc line detailing those involved in its writing" />
+</figure>
+
+<p>However, internal documents sometimes display a portion of the work women performed in computing.
+    In the case of the choices made for which symbols computers would use, this archive demonstrates that women were heavily involved.
+    In a memo submitted to Laura by Edna Verzuh, there is a page of symbols which appear to await her commentary.
+    This is precedented by the file following this memo, in which there are comments on the symbols requested by Edna herself, which are marked by both her name on the sheet and her initials, ETV, in the bottom right-hand corner, as well as comments by Nanny, marked also by her name and her initials, NB, on the bottom left-hand corner. This Nanny is also likely Frank Verzuh’s secretary, seeing as the secretary subscript on most of Verzuh’s outgoing correspondence is nb.
+    This is one example of the many tasks secretaries completed which diverged from the tasks we typically associate with them, and portrays how secretaries were and are far more than typists and organizers.</p>
+
+<p>After the memo was received by Laura, final comments seem to have been sent by her to Frank Verzuh in a note requesting the agreed upon new symbols, although the initials on the bottom of this note are too ambiguous to say with certainty.</p>
+
+    <p>Through these documents, we are able to view a piece of the role women played in the administrative details regarding computing, as well as some of the decision making and female labor behind the MIT Computation Center.
+        Secretaries, which have often been credited only as typists, were often responsible for work relying on their judgement and decision making as well.
+        This trend is also present in scholarly manuscripts, as shown in <i>#ThanksForTyping Spotlights Unnamed Women In Literary Acknowledgments</i>, an NPR article discussing the female labor behind many scholarly publications.
+        This article provides evidence of academics’ wives typing up manuscripts and doing research they were often unnamed for, being credited frequently as simple “my wife,” parallel to the secretary being recognized solely by initials in external communications.</p>
+
+<a href="/archives/doc/553">File with Edna Verzuh's and Nanny B.'s notes</a>
+    <p></p>
+    <a href="/archives/doc/552">Memo to Laura</a>
+    <p></p>
+    <a href="/archives/doc/551">Final note to Frank Verzuh</a>
+    <p></p>
+    <a href="https://www.npr.org/2017/03/30/521931310/-thanksfortyping-spotlights-unnamed-women
```

Comment:
I think it's ok to keep the link only in-line above since you are only pointing to archival documents above.